src/router/index.ts

```diff
@@ -18,6 +18,11 @@ const router = createRouter({
         // dialog: Maps,
       },
     },
+    {
```

Comment:
I think you have used this in place of loader they what is the purpose of adding them as a separate routes.